src/libs/_tests/runtime-utils.test.js

```diff
@@ -74,7 +76,52 @@ const galaxyDisk = {
   diskType: 'pd-standard',
   googleProject: 'terra-test-e4000484',
   id: 10,
-  labels: { saturnApplication: 'galaxy' }, // Note 'galaxy' vs. 'GALAXY', to represent our older naming scheme
+  labels: { saturnApplication: 'galaxy', saturnWorkspaceName: 'test-workspace' }, // Note 'galaxy' vs. 'GALAXY', to represent our older naming scheme
+  name: 'saturn-pd-026594ac-d829-423d-a8df-76fe96f5b4e7',
+  size: 500,
+  status: 'Ready',
+  zone: 'us-central1-a'
+}
+
+const galaxyDisk2 = {
+  auditInfo: {
+    creator: 'cahrens@gmail.com', createdDate: '2021-11-29T20:19:13.162484Z', destroyedDate: null, dateAccessed: '2021-11-29T20:19:14.114Z'
+  },
+  blockSize: 4096,
+  diskType: 'pd-standard',
+  googleProject: 'terra-test-e4000484-2',
+  id: 10,
+  labels: { saturnApplication: 'galaxy', saturnWorkspaceName: 'test-workspace' }, // Note 'galaxy' vs. 'GALAXY', to represent our older naming scheme
```

Comment:
In general, `saturnApplication` should be "GALAXY". I had one with "galaxy" to represent a certain condition (which is why I had the comment after it), but we don't want to litter that all of the place because it makes it more confusing.